src/components/TribeMember/TribeMemberFront.js

```diff
@@ -31,15 +31,8 @@ const TribeMemberFront = ({
   onClick,
   sepColour,
 }) => {
-  const [hasMouseOver, setHasMouseOver] = React.useState(false)
-
   return (
-    <figure
-      className="tribe-figure"
-      onClick={onClick}
-      onMouseOver={() => setHasMouseOver(true)}
```

Comment:
Were these mouse over attrs not doing anything?

Reply:
They were setting a boolean into state, which wasn't read anywhere.